images/macos/provision/core/android-toolsets.sh

```diff
@@ -48,17 +66,14 @@ echo y | $SDKMANAGER "cmake;3.6.4111459"
 echo "Installing latest ndk..."
 echo y | $SDKMANAGER "ndk-bundle"
 
-for platform_name in "${ANDROID_PLATFORM_LIST[@]}"
-do
-    echo "Installing platform $platform_name ..."
-    echo y | $SDKMANAGER "platforms;$platform_name"
-done
+availablePlatforms=($(${ANDROID_HOME}/tools/bin/sdkmanager --list | sed -n '/Available Packages:/,/^$/p' | grep "platforms;android-" | cut -d"|" -f 1))
```

Comment:
Use the same approach as you use for allBuildTools without sed

Reply:
Done.